src/components/tabs/themes/tabs.base.scss

```diff
@@ -1,116 +1,155 @@
 @use 'styles/common/component';
 @use 'styles/utilities' as *;
-
-$tabs-animation-function: cubic-bezier(.35, 0, .25, 1);
+@use 'tabs.common' as *;
+@use 'sass:map';
 
 :host {
-    display: flex;
-    flex-direction: column;
+    // Scroll buttons
+    --scroll-btn-size: #{rem(48px)};
+    --tabs-width: 100%;
+    --tab-max-width: #{rem(360px)};
+    --tab-min-width: #{rem(90px)};
+    --_border-size: #{rem(1px)};
+
+    display: block;
     overflow: hidden;
+    position: relative;
+    height: 100%;
 }
 
-:host([alignment='start']) [part='headers-scroll'] {
-    ::slotted(igc-tab:last-of-type) {
-        margin-inline-end: auto;
-    }
+%fake-column {
+    content: '';
+    width: 100%;
+    height: 100%;
+    grid-row: 1;
 }
 
-:host([alignment='end']) [part='headers-scroll'] {
-    ::slotted(igc-tab:first-of-type) {
-        margin-inline-start: auto;
-    }
-}
+[part~='tabs'] {
+    display: flex;
+    height: 100%;
+    scroll-padding-inline: var(--scroll-btn-size);
+    scroll-behavior: smooth;
+    overflow: scroll hidden;
+    scroll-snap-type: both proximity;
 
-:host([alignment='center']) [part='headers-scroll'] {
-    ::slotted(igc-tab:first-of-type) {
-        margin-inline-start: auto;
-    }
+    // Hide the scrollbar for mobile devices
 
-    ::slotted(igc-tab:last-of-type) {
-        margin-inline-end: auto;
+    scrollbar-width: none;
+    overscroll-behavior: none;
+
+    &::-webkit-scrollbar {
+        display: none;
     }
 }
 
-:host([alignment='justify']) [part='headers-scroll'] {
+[part~='inner'] {
+    display: grid;
+    grid-template-rows: minmax(var(--header-min-height), auto) auto 1fr;
+    align-items: center;
+    position: relative;
+    flex-grow: 1;
+
     ::slotted(igc-tab) {
-        flex-basis: 100px;
-        flex-grow: 1;
-        max-width: 100%;
+        display: contents;
     }
 }
 
-[part='headers'] {
-    display: flex;
-    overflow: hidden;
-    min-height: rem(42px);
-    flex: none;
+[part~='scrollable'] {
+    /* stylelint-disable */
```

Comment:
What stylelint rule are you disabling here? When possible disable only specific rules and only if specifically needed.